### PR TITLE
[Easy Merge] Add Mapping to File Mapper

### LINKED ISSUE #39: Register directory mapping with File Mapper
This mapping is very useful as in most cases this will handle most scenarios

---

https://github.com/googleinterns/filesystem-in-appsscript/blob/757fea2611814333eb46f48fe5c52b87a0c5ce21/FileSystem/Workbook.js#L101-L106

---

###### This issue was generated by [todo](https://todo.jasonet.co) based on a `todo` comment in 757fea2611814333eb46f48fe5c52b87a0c5ce21 when #22 was merged. cc @googleinterns.

### PR DESCRIPTION
**Pull request by satviksr**
Fixes #39 

Adds mapping to config when active workbook path is set.

### CODE REVIEW

File: FileSystem/FileMapperMocker.js [hunk 1]
```diff
@@ -48,10 +48,11 @@ var VBAFileMapperMocker = {
   getRelativePathSplit: getRelativePathSplit,
   getBaseDriveFolder: getBaseDriveFolder,
   addFileMapping: addFileMapping,
-  addFolderMapping: addFolderMapping
+  addFolderMapping: addFolderMapping,
+  clearAllMappingsInConfig: clearAllMappingsInConfig,
 };
 
-var FileMapper = USE_FILEMAPPER_MOCKER ? VBAFileMapperMocker : VBAFileMapper;
```
Comment (paneendra-google) [inline FileSystem/FileMapperMocker.js:54]: Prefer VBA over Vba

File: FileSystem/appsscript.json [hunk 2]
```diff
@@ -1,18 +1,20 @@
...
+    }]
   },
   "webapp": {
     "access": "DOMAIN",
     "executeAs": "USER_DEPLOYING"
   },
   "exceptionLogging": "STACKDRIVER",
   "runtimeVersion": "DEPRECATED_ES5"
-}
+}
```
Comment (paneendra-google) [inline FileSystem/appsscript.json]: add newline at EOF

**Review by paneendra-google — CHANGES_REQUESTED** [review]

File: FileSystem/FileMapperMocker.js (hunk 1 above)
Comment (satviksr) [reply FileSystem/FileMapperMocker.js:54]: Fixed

File: FileSystem/appsscript.json (hunk 2 above)
Comment (satviksr) [reply FileSystem/appsscript.json]: Fixed

**Review by paneendra-google — APPROVED** [review]

**Review by monikadhok — APPROVED** [review]

**Pull request merged**